### DISCUSSION TITLE
Is this repo still maintained?

### DISCUSSION THREAD

**Discussion by weitang2022** [OpenTalker/SadTalker#922] [General]
I don't see new releases since June 2023.
Anyone had luck to make it work on Mac with M1 chip and Python 3.11?

**Comment by acetopwon** [comment]
I haven't had much luck considering a lot of the libraries are closely related to CUDA which is used for machines with nvidia graphics card (correct me if I'm wrong, as I am not much far advanced than most of the people) so I had to SSH into a server and run it there.

**Comment by fuqifacai** [comment]
Seems not maintained .

**Comment by MalikTayyabTanveer** [comment]
> I don't see new releases since June 2023. Anyone had luck to make it work on Mac with M1 chip and Python 3.11?

Yes, I ran it on my M1.
